modules/cron/tasks_extended.go

```diff
@@ -131,6 +131,25 @@ func registerDeleteOldActions() {
 	})
 }
 
+func registerUpdateGiteaChecker() {
+	type UpdateCheckerConfig struct {
+		BaseConfig
+		Timeout      time.Duration
+		HTTPEndpoint string
+	}
+	RegisterTaskFatal("update_checker", &UpdateCheckerConfig{
+		BaseConfig: BaseConfig{
+			Enabled:    false,
+			RunAtStart: false,
+			Schedule:   "@every 168h",
```

Comment:
If the last run time is not saved in database, and without `RunAtStart`, users may not have a chance to check updates if the Gitea server is restarted periodically.

Reply:
Well, that's their own problem I'd say.
If you restart your server often enough not to trigger the cron job, then that's on you.

Reply:
Yeah, there are other weekly tasks as well.